pkg/webui/oauth/views/update-password/index.js

```diff
@@ -0,0 +1,230 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { Container, Col, Row } from 'react-grid-system'
+import { Redirect } from 'react-router-dom'
+import bind from 'autobind-decorator'
+import { defineMessages } from 'react-intl'
+import { push } from 'connected-react-router'
+import { connect } from 'react-redux'
+import * as Yup from 'yup'
+import queryString from 'query-string'
+
+import api from '../../api'
+import sharedMessages from '../../../lib/shared-messages'
+import { digit, uppercase, special } from '../../lib/regexp'
+
+import Button from '../../../components/button'
+import Form from '../../../components/form'
+import Input from '../../../components/input'
+import SubmitButton from '../../../components/submit-button'
+import IntlHelmet from '../../../lib/components/intl-helmet'
+import Message from '../../../lib/components/message'
+import style from '../create-account/create-account.styl'
+import Checkbox from '../../../components/checkbox'
+import Spinner from '../../../components/spinner'
+import PropTypes from '../../../lib/prop-types'
+
+const m = defineMessages({
+  newPassword: 'New Password',
+  oldPassword: 'Temporary or Old Password',
```

Comment:
```suggestion
  oldPassword: 'Temporary or Old Password',
```
Let's not mention the temporary password here. If the user obtained a temporary password, he/she was not logged in and will use the other update password flow. This might just generate confusion.